rotkehlchen/globaldb/asset_updates/manager.py

```diff
@@ -194,9 +194,12 @@ def __init__(self, msg_aggregator: 'MessagesAggregator', globaldb: 'GlobalDBHand
         self.asset_parser = AssetParser()
         self.asset_collection_parser = AssetCollectionParser()
         self.multiasset_mappings_parser = MultiAssetMappingsParser()
-        self.branch = os.getenv('GITHUB_BASE_REF', 'develop')
         if is_production():
             self.branch = 'master'
+        elif (base_ref := os.getenv('GITHUB_BASE_REF', 'develop')) != '':
+            self.branch = base_ref
+        else:  # In the CI when it's not triggered by a PR the base ref is ''
```

Comment:
> The GITHUB_BASE_REF is '' in the CI when NOT running from a PR. This was causing the url in the asset updater to be https://raw.githubusercontent.com/rotki/assets//updates/info.json (missing the branch). Apparently until recently github was resolving this to the default branch, but that now returns 404. So changed it to default to develop in this case. We already do something similar here: https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/conftest.py#L401

When did this change? Won't this affect other places in our CI too? Say builds? We had a succesfull nightlies run in your last fix. That's why I am asking.